src/chrome/content/rules/Eldarya.com.xml

```diff
@@ -0,0 +1,15 @@
+<!--
+	Nonfunctional hosts in *.eldarya.com:
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Eldarya.com">
+	<target host="eldarya.com" />
+	<target host="www.eldarya.com" />
```

Comment:
You should have a look at https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#enumerating-subdomains